application/wypl-core/src/main/java/com/wypl/wyplcore/schedule/controller/ScheduleController.java

```diff
@@ -19,11 +19,12 @@
 @RequestMapping("/schedule/v2/schedules")
 public class ScheduleController {
 
-    private final ScheduleService scheduleService;
+	private final ScheduleService scheduleService;
 
-    @PostMapping
-    public WyplResponseEntity<ScheduleInfoCreateResponse> addSchedule(@Authenticated AuthMember authMember, @RequestBody ScheduleCreateRequest scheduleCreateRequest) {
-        ScheduleInfoCreateResponse response = scheduleService.createSchedule(authMember, scheduleCreateRequest);
-        return WyplResponseEntity.created(response, "일정이 생성됐습니다.");
-    }
+	@PostMapping
+	public WyplResponseEntity<ScheduleInfoCreateResponse> addSchedule(@Authenticated AuthMember authMember,
+		@RequestBody ScheduleCreateRequest scheduleCreateRequest) {
```

Comment:
해당 내용도 위에는 있지만!

```suggestion
	public WyplResponseEntity<ScheduleInfoCreateResponse> addSchedule(
		@Authenticated AuthMember authMember,
		@RequestBody ScheduleCreateRequest scheduleCreateRequest
	) {
```
와 같이 수정하는게 더 보기 편하다고 생각하는데 어떠신가요?